src/components/ui/coaching-sessions/coaching-session-title.tsx

```diff
@@ -49,10 +55,40 @@ const CoachingSessionTitle: React.FC<{
 
   const displayTitle = sessionTitle?.title || defaultSessionTitle().title;
 
+  // Helper to get presence by role
+  const getPresenceByRole = (role: 'coach' | 'coachee'): UserPresence | undefined => {
+    if (!presenceState) return undefined;
+    return Array.from(presenceState.users.values()).find(u => u.role === role);
+  };
+
+  // Enhanced title rendering with presence indicators
+  const renderTitleWithPresence = () => {
+    if (!sessionTitle || !currentCoachingRelationship) return displayTitle;
+    
+    const coachPresence = getPresenceByRole('coach');
+    const coacheePresence = getPresenceByRole('coachee');
+    
+    // Parse the existing title format: "Coach Name <> Coachee Name"
+    const parts = displayTitle.split(' <> ');
+    if (parts.length !== 2) return displayTitle;
+    
+    const [coachName, coacheeName] = parts;
+    
+    return (
+      <span className="flex items-center gap-1">
+        <PresenceIndicator presence={coachPresence} />
+        <span>{coachName}</span>
+        <span className="mx-2">&lt;&gt;</span>
```

Comment:
Let's switch the separator to a '/' character HTML-encoded as `&#47;`

Reply:
Updated d1b63a85d76e655410fe3721f3bf917280804484